doc/guide/bindings.hpp

```diff
@@ -865,15 +865,15 @@ The code for the command-line bindings is found in @c src/mlpack/bindings/cli.
 @subsection bindings_cli_mlpack_main mlpackMain() definition
 
 Any command-line program must be compiled with the @c BINDING_TYPE macro
-set to the value @c BINDING_TYPE_CLI.  This is handled by the CMake macro
+set to the value @c BINDING_TYPE_IO.  This is handled by the CMake macro
```

Comment:
```suggestion
set to the value @c BINDING_TYPE_CLI.  This is handled by the CMake macro
```
Hi @shrit, I am not sure, but I think so there some more `IO` which are not replaced correctly in `doc/` folder.

Reply:
Exactly, thanks for the review